charts/kong/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 * Added serviceMonitor.trustCRDsExist for render based deployments
+* Add migrations.ttlSecondsAfterFinished option to chart
```

Comment:
```suggestion
* Add `migrations.ttlSecondsAfterFinished` option to chart
  [#1174](https://github.com/Kong/charts/pull/1174)
```

Reply:
@pmalek I have updated the files accordingly.